projects/odf_transform/odf_transform/odf_parser/odf_parser.py

```diff
@@ -6,12 +11,23 @@
 import gsw
 
 # Dictionary with the mapping of the odf types to python types
-odf_dtypes = {'DOUB': 'float64', 'SING': 'float32', 'DOUBLE': 'float64',
-              'SYTM': str, 'INTE': 'int32', 'CHAR': str, 'QQQQ': 'int32'}
+odf_dtypes = {
+    'DOUB': 'float64',
+    'SING': 'float32',
+    'DOUBLE': 'float64',
+    'SYTM': str,
+    'INTE': 'int32',
+    'CHAR': str,
+    'QQQQ': 'int32'
+}
 
 # Commonly date place holder used within the ODF files
-odf_time_null_value = (dt.datetime.strptime("17-NOV-1858 00:00:00.00", '%d-%b-%Y %H:%M:%S.%f') -
-                       dt.datetime(1970, 1, 1)).total_seconds()
+odf_time_null_value = (
+        dt.datetime.strptime("17-NOV-1858 00:00:00.00", '%d-%b-%Y %H:%M:%S.%f') - dt.datetime(1970, 1, 1)
```

Comment:
is this just a contant number thats in the ODF files?

Reply:
yes it seems to be a placeholder for time values in the header, and also in the time variables.

Reply:
the place holder is  "`17-NOV-1858 00:00:00.00"` but converted to unix time it becomes `-3506716800.0`

Reply:
17-NOV-1858 00:00:00.00 is the fill value for missing dates

Reply:
can we just put `17-NOV-1858 00:00:00.00` as an NA value in `read_csv`? Or is that done already

Reply:
it is done already but it this refer more the time values available within the header which gets converted to variables.